conf/distro/bisdn-linux.conf

```diff
@@ -29,7 +29,7 @@ PREFERRED_PROVIDER_udev ?= "systemd"
 PREFERRED_PROVIDER_udev-utils ?= "systemd"
 
 BAD_RECOMMENDATIONS ?= "busybox-syslog"
-DISTRO_FEATURES_remove += "alsa"
+DISTRO_FEATURES_remove = " alsa"
```

Comment:
If I understand the documentation correctly, the space prefix/suffix is only for append/prepend. No space for remove. The commit should be fixed as well.

Reply:
Weird, I am sure that when I grepped for it, I found plenty of examples in Yocto. But now I can't find them anymore. Must have dreamt then.